.bandit

```diff
@@ -1,3 +1,3 @@
 [bandit]
-skips: B303
-exclude: frontend\webapp\static
+skips: B301,B101
```

Comment:
this skips warnings about pickle and assertions (B303 was removed because it was for MD5)